app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
The `Dwarf` class should be abstract since it does not implement all abstract methods from the `Player` class.

Reply:
Why this not fixed ? Is Dwarf is not abstract class ? Since it has abstract method  play_elf_song and passes test "test_main.py::test_classes_should_be_abstract[Dwarf] PASSED              [ 26%]"

![image](https://github.com/mate-academy/py-elves-and-dwarves/assets/61278036/80a424d1-6928-449d-9aae-f55f16678eb4)